tests/shared/src/main/scala/munit/ScalaCheckFrameworkSuite.scala

```diff
@@ -7,7 +7,7 @@ class ScalaCheckFrameworkSuite extends ScalaCheckSuite {
 
   // NOTE(gabro): this is needed for making the test output stable for the failed test below.
   // It also serves as a test for overriding these parameters.
-  override def scalaCheckTestParameters =
+  override def scalaCheckTestParameters: org.scalacheck.Test.Parameters =
```

Comment:
was this required by Scalafix? I must have missed it

Reply:
anyway, if you rebase on master that's gone now

Reply:
Yep